### ISSUE TITLE
High Performance Now Playing Updates Issue

### ISSUE THREAD

**Issue by metacast1** [AzuraCast/AzuraCast#5915]
### Installation Method

One-Click Installation (Docker)

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

0.17.6

### What happened?

We use an event driven AWS lambda function to pull now playing metadata. This function is not designed to maintain continuous connections, and the new high performance now playing update will cause issues because there is no event to send to our backend.

Do you have any suggestions to send an event rather than maintaining a continuous connection?

**Comment by BusterNeece** [comment]
@metacast1 I don't know quite what you mean. The "high-performance" Now Playing updates are just an option for people who want to take advantage of them, and they are an inline replacement for our previous NChan library. You can still just poll our existing Now Playing API endpoints for changes, or set up a Web Hook to push changes out to other servers upon certain events.

**Comment by fafamonge** [comment]
I'm not shure if it's related to but I'm getting 502 Bad Gateway error message, the CPU's consumption on sevrer is 100%. When I go to docker-compose logs -f web:

 AzuraCast.WARNING: Console command `azuracast:sync:nowplaying` exited with error code 1054. [] []
azuracast  |
azuracast  | In ExceptionConverter.php line 67:
azuracast  |
azuracast  |   An exception occurred while executing a query: SQLSTATE[42S22]: Column not
azuracast  |   found: 1054 Unknown column 't0.enable_static_nowplaying' in 'field list'
azuracast  |
azuracast  |
azuracast  | In Exception.php line 28:
azuracast  |
azuracast  |   SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.enable_static_no
azuracast  |   wplaying' in 'field list'
azuracast  |
azuracast  |
azuracast  | In Connection.php line 69:
azuracast  |
azuracast  |   SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.enable_static_no
azuracast  |   wplaying' in 'field list'
azuracast  |
azuracast  |
azuracast  | azuracast:sync:nowplaying [-t|--timeout [TIMEOUT]]

Stations are not able to start.

**Comment by metacast1** [comment]
Thanks Buster, I did not understand that we could still use the old method.

**Comment by BusterNeece** [comment]
@metacast1 Definitely! We just want to have more ways of accessing the Now Playing APIs, for various use-cases.
